client/my-sites/checkout/checkout-thank-you/redesign-v2/is-redesign-v2.ts

```diff
@@ -16,6 +16,11 @@ const isRedesignV2 = ( props: CheckoutThankYouCombinedProps ) => {
 
 	// ThankYou page for only purchasing a plan.
 	const purchases = getPurchases( props );
+
+	if ( purchases.every( isDomainTransfer ) ) {
```

Comment:
Is there a possibility for the domain transfer flow to include other products in the checkout cart? E.g., themes/plugins/plans/etc.

Curious but not a blocker to merging this.

Reply:
It shouldn't be possible I think! But I'm not sure about this.